pages/common/sass.md

```diff
@@ -0,0 +1,20 @@
+# Sass
+
+> Converts SCSS or Sass files to CSS
+> .scss and .sass can be used interchangeably in below examples depending on your preference
```

Comment:
We shouldn't be 'picking sides', but I don't think it's useful to add a note about the conventional file extensions. Especially because they're not required I believe?